posthog/api/decide.py

```diff
@@ -145,6 +146,8 @@ def get_decide(request: HttpRequest):
             team = user.teams.get(id=project_id)
 
         if team:
+            structlog.contextvars.bind_contextvars(team_id=team.id)
```

Comment:
same makes sense for `capture` ?

https://github.com/PostHog/posthog/blob/master/posthog/api/capture.py#L208 - ingestion_capture there resolves the teamID

Reply:
yes but I'll leave that to someone else, as an exercise for the reader 😏